### PR TITLE
Add Thanks page to redirect people to after donating

### LINKED ISSUE #27: Need a "Thank you" page to redirect people to after donations
"Thank you for your support. Every little bit counts to help us work towards our initiative and keep these workshops running."

### PR DESCRIPTION
**Pull request by conchan**
Resolves #27 

thanks

### CODE REVIEW

**Comment by conchan** [comment]
thanks2

File: source/thanks.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Thank you! - RailsBridge Boston
+---
+
+<main>
+  <h1>
+    Thanks!
+    <%= image_tag "heart.svg", class: "thanks-heart" %>
+  </h1>
+  <p>Thank you for your support. Every little bit counts to help us work towards our initiative and keep these workshops running.
```
Comment (mxie) [inline source/thanks.html.erb]: Can we move the copy into the next link?

**Comment by mxie** [comment]
Just 1 small comment. Looks good otherwise.

**Comment by mborsare** [comment]
LGTM, nice touch!

**Pull request merged**